app/views/inthenews/search_events.html.erb

```diff
@@ -1,17 +1,8 @@
-<h1 class="sectiontitle"><%= t "inthenews.title" %></h1>
-
-<h2>Events Search</h2>
-
-<%= will_paginate @items %>
-<%= pluralize @items.count, "Event" %>
-
-<ul>
-  <% @items.each do |item| %>
-    <li>
-      <%= link_to item.name, inthenews_event_path(item) %>
-      | <%= item.event_type.name %>
-    </li>
-  <% end %>
-</ul>
-
-<%= will_paginate @items %>
+<%# NOTE: apparently have to use layout not "partial" if passing a block %>
```

Comment:
Cool beans! Hadn't seen this before: https://guides.rubyonrails.org/v5.2/action_view_overview.html#partial-layouts

Reply:
I ended up having to refer back to the marching band project, because I recalled running into headaches trying to use blocks with partials there as well!  It's a neat trick.